packages/material-ui/src/TextField/TextField.js

```diff
@@ -230,15 +234,18 @@ TextField.propTypes = {
    */
   InputLabelProps: PropTypes.object,
   /**
-   * Properties applied to the `Input` element.
+   * Properties applied to the Input element.
+   * It will be a [`FilledInput`](/api/filled-input/),
+   * [`OutlinedInput`](/api/outlined-input/) or [`Input`](/api/input/)
+   * component depending on the `variant` prop value.
    */
   InputProps: PropTypes.object,
   /**
-   * Attributes applied to the native `input` element.
+   * Attributes applied to the `input` element.
    */
   inputProps: PropTypes.object,
   /**
-   * Use this property to pass a ref callback to the native input component.
+   * Use this property to pass a ref callback to the `input` element.
```

Comment:
Needs to be consistent with whichever option is chosen for the previous examples.

Here's option 2 if needed:
```suggestion
   * This property can be used to pass a ref callback to the `input` element.
```